packages/wdio-spec-reporter/src/index.js

```diff
@@ -387,7 +387,7 @@ class SpecReporter extends WDIOReporter {
     getEnviromentCombo (caps, verbose = true, isMultiremote = false) {
         const device = caps.deviceName
         const browser = isMultiremote ? 'MultiremoteBrowser' : (caps.browserName || caps.browser)
-        const version = caps.version || caps.platformVersion || caps.browser_version
+        const version = caps.browserVersion || caps.browser_version || caps.version || caps.platformVersion
```

Comment:
There are two issues with this:

1. `browser_version` is not a valid capability
2. we are mixing platform version and browser version here

Reply:
1) I found usage `browser_version` in browserstack. When no version specified in configs - BS gives latest and retunrs `browser_version` rather then browserVersion. So I moved it back
2) `platformVersion` version is usually set for mobile devices.
![image](https://user-images.githubusercontent.com/6696821/81657235-631ec480-9440-11ea-9d0e-0d7a220791f8.png)![image](https://user-images.githubusercontent.com/6696821/81657291-6f0a8680-9440-11ea-82e0-5644c25a95b8.png)
So I don't think removal of platformVersion is required

Reply:
> I found usage browser_version in browserstack

BS is using invalid capabilities which I don't know why. I am fine with adding some extra logic to accomodate BS users but we should add a comment mentioning that this only for this use case. 

I think it makes sense to check if it is a mobile session and change the structure accordingly.

Reply:
Yes, everybody (SauceLabs, TestingBot, Crossbrowsertesting) use `version`.
Updated order to `caps.version || caps.browserVersion || caps.browser_version`